themes/aframe/source/css/_settings.styl

```diff
@@ -47,6 +47,12 @@ media--mobile-device() {
   }
 }
 
+media--mobile-only() {
```

Comment:
Possible to not use a mobile-specific `max-width` media query? I generally only do `min-width` because it's better for responsiveness. If you do `max-width`, then we are sort of splitting up the site into two different parts.